tutorials/clinical/10_ieeg_localize.py

```diff
@@ -192,27 +192,28 @@ def plot_overlay(image, compare, title, thresh=None):
 del CT_orig
 
 # %%
-# Alignment failures sometimes occur which requires manual alignment.
-#
-# This can be done using Freesurfer's ``freeview`` to align manually
-#     - Load the two scans from the command line:
-#       ``freeview $MISC_PATH/seeg/sample_seeg/mri/T1.mgz
-#       $MISC_PATH/seeg/sample_seeg_CT.mgz``
-#     - Navigate to the upper toolbar, go to ``Tools>>Transform Volume...``
-#     - Use the rotation and translation slide bars to align the CT
-#       to the MR (be sure to have the CT selected in the upper left menu)
-#     - Save the modified volume using the ``Save Volume As...`` button
-#     - Resample to the T1 shape and affine using::
-#
-#           CT_aligned_pre = nib.load(op.join(misc_path, 'seeg',
-#                                             'sample_seeg_CT_aligned.mgz'))
-#           CT_aligned = resample(moving=np.asarray(CT_aligned_pre.dataobj),
-#                                 static=np.asarray(T1.dataobj),
-#                                 moving_affine=CT_aligned_pre.affine,
-#                                 static_affine=T1.affine)
-#
-#    The rest of the tutorial can then be completed using ``CT_aligned``
-#    from this point on.
+# .. note::
```

Comment:
Have you tried locally `.. sidebar::` instead? You can do for example:
```
PATTERN=10_ieeg make html_dev-pattern
```
and you should be able to look at the output to see if it's better or worse

Reply:
... though I guess this is an entire section. It's a bit weird to put an entire section in a note, no?

Reply:
It actually doesn't need to be executed and it doesn't look good, it ends up looking out of place